services/discordService.js

```diff
@@ -52,7 +52,31 @@ const setInDiscordFalseScript = async () => {
   await Promise.all(updateUsersPromises);
 };
 
+const generateAuthTokenForCloudflare = async () => {
```

Comment:
where did you use authToken, that means 403 error

Reply:
It is being consumed in the function that makes the fetch call just below this function